### PR TITLE
converted stateless lifecycleless class components to stateless funct…

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
## Issue
[Convert stateless components to functional component #518](https://github.com/infinitered/reactotron/issues/518)

## Work done
 converted stateless lifecycleless class components to stateless functional components

### CODE REVIEW

File: packages/reactotron-app/App/Shared/Timestamp.js [hunk 1]
```diff
@@ -16,25 +16,24 @@ const Styles = {
...
-  }
+  return (
+    <span style={containerStyles}>
+      <span style={Styles.left}>{left}</span>
+      <span style={Styles.right}>{right}</span>
+    </span>
+  )
+}
+
+Timestamp.ropTypes = {
```
Comment (skellock) [inline packages/reactotron-app/App/Shared/Timestamp.js]: Oopsie. `propTypes`.

**Comment by M-ZubairAhmed** [comment]
oh so sorry for that will do that first thing tomorrow

**Comment by M-ZubairAhmed** [comment]
i just fixed the typo and updated the PR, please take a look

**Comment by skellock** [comment]
Wonderful, thank you kindly!

**Pull request merged**